pylops/medical/ct.py

```diff
@@ -50,7 +52,11 @@ class CT2D(LinearOperator):
         Distance between origin and detector along the source-origin line
         (only for "proj_geom_type=fanflat")
     projector_type : :obj:`int`, optional
-        Type of projection geometry (``strip``, or ``line``, or ``linear``)
+        Type of projection kernel (``strip``, or ``line``, or ``linear``).
+        For ``cuda`` computation engine, only ``cuda`` (hardware-accelerated ``linear``)
```

Comment:
Type of projection geometry: ``strip``, or ``line``, or ``linear`` for ``engine=cuda``,
and  ``cuda`` (i.e., hardware-accelerated ``linear``) for ``engine=cuda``.

Reply:
Adjusted the wording as requested, but I suggest changing "geometry" -> "kernel" (or a better word?), because to me the geometry is more about it being parallel or fanbeam, etc.

Reply:
Yes sorry, I thought I would push one bigger batch, but testing some things is taking more time than expected :)